src/stan-users-guide/parallelization.Rmd

```diff
@@ -687,3 +699,62 @@ function if they are rectangular.
 If the outputs are of varying sizes, then there will have to be some way
 to convert it back to a usable form based on the input, because there
 is no way to directly return sizes or a ragged structure.
+
+## OpenCL
+
+OpenCL (Open Computing Language) is a framework that enables writing programs that
+execute across heterogeneous platforms. An OpenCL program can be run on CPUs, GPUs,
+DSPs or FPGAs. In order to run OpenCL programs, an OpenCL runtime be installed on
+they target system.
+
+Stan's OpenCL backend is currently supported in CmdStan and its wrappers. In order
+to use it, the model must be compiled with the `STAN_OPENCL` makefile flag. Setting
+this flag means that the Stan-to-C++ translator (`stanc3`) will be supplied the
+`--use-opencl` flag and that the OpenCL enabled backend (Stan Math functions) will be enabled.
+
+In Stan, the following distributions can be automatically run in parallel on both CPUs
+and GPUs with OpenCL:
+
+- bernoulli_lpmf
+- bernoulli_logit_lpmf
+- bernoulli_logit_glm_lpmf*
+- beta_lpdf
+- beta_proportion_lpdf
+- binomial_lpmf
+- categorical_logit_glm_lpmf*
+- cauchy_lpdf
+- chi_square_lpdf
+- double_exponential_lpdf
+- exp_mod_normal_lpdf
+- exponential_lpdf
+- frechet_lpdf
+- gamma_lpdf
+- gumbel_lpdf
+- inv_chi_square_lpdf
+- inv_gamma_lpdf
+- logistic_lpdf
+- lognormal_lpdf
+- neg_binomial_lpmf
+- neg_binomial_2_lpmf
+- neg_binomial_2_log_lpmf
+- neg_binomial_2_log_glm_lpmf*
+- normal_lpdf
+- normal_id_glm_lpdf*
+- ordered_logistic_glm_lpmf*
+- pareto_lpdf
+- pareto_type_2_lpdf
+- poisson_lpmf
+- poisson_log_lpmf
+- poisson_log_glm_lpmf*
+- rayleigh_lpdf
+- scaled_inv_chi_square_lpdf
+- skew_normal_lpdf
+- std_normal_lpdf
+- student_t_lpdf
+- uniform_lpdf
+- weibull_lpdf
+
+* GLM functions are used in cases where X is not a `row_vector`.
```

Comment:
Is this still true?

Reply:
Yes.

Reply:
What does this bit mean though?

Reply:
Like:

```
ordered_logistic_glm_lpmf*
```

GLM functions are used... But of course they're used, it's the glm function!

Reply:
With OpenCL is missing.